packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -551,6 +551,71 @@ class NavigatorObserver {
 /// because it doesn't depend on its animation parameters (elided with `_`
 /// and `__` in this example). The transition is built on every frame
 /// for its duration.
+///
+/// ### Embedding Navigators
+///
+/// An app can use more than one Navigator. Embedding one Navigator below
+/// another Navigator can result in an "inner journey" such as user
+/// registration, store checkout, or other independent journeys that represent
+/// a subsection of your overall application.
+///
+/// ```dart
```

Comment:
Maybe move this below the caveat below. Otherwise, it might seem like you're giving an examples of how to do it whereas you're giving an example of a mistake.

Reply:
@xster I didn't follow that.  The code example here is an example of how to nest Navigators.  It's not an example of a mistake...